var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -111,3 +115,34 @@ def install(self, spec, prefix):
         # if self.run_tests:
         #     make('test')            # 'VERBOSE=1'
         make('install')
+
+    @run_after('install')
+    def link_system_certs(self):
+        if '+systemcerts' not in self.spec:
+            return
+
+        system_dirs = [
+            # CentOS, Fedora, RHEL
+            '/etc/pki/tls',
+            # Ubuntu
+            '/usr/lib/ssl'
+        ]
+
+        pkg_dir = join_path(self.prefix, 'etc', 'openssl')
+
+        for dir in system_dirs:
```

Comment:
Minor note: can we use a different name, like `directory`? Asking just because [dir](https://docs.python.org/3/library/functions.html#dir) is a built-in function in python.

Reply:
I changed it to `directory` as you proposed.